rf-stylez.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'rubocop', '>= 0.59', '< 0.81'
   spec.add_runtime_dependency 'rubocop-rails', '~> 2.5.0'
+  spec.add_runtime_dependency 'get_env', '~> 0.2'
```

Comment:
The build failed with `Bundler could not find compatible versions for gem "get_env"` - does it need a `gem install get_env` since it's a runtime dep?

Reply:
Nope, this should work just fine once `get_env` is released to rubygems, `add_runtime_dependency` is an alias to `add_dependency`